stories/text.js

```diff
@@ -0,0 +1,35 @@
+import _ from 'lodash';
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs/react';
+import { withInfo } from '@storybook/addon-info';
+
+import { knobs, getKnobTypeForProp } from './knobs/text';
+
+import Text from '../src/components/text';
+
+const knobText = {
+    label: 'Child text',
+    defaultValue: 'We Love EyeEm'
+}
+
+storiesOf('Text', module)
+    .addDecorator((story, context) => withInfo('')(story)(context))
+    .addDecorator(withKnobs)
+    .add('with controllable props', () => {
+        const props = _.reduce(
```

Comment:
Same - this can now be moved into a helper function to be re-used across all stories that generate knobs based on props